cmake/cfg_hash.cmake

```diff
@@ -0,0 +1,8 @@
+if (UNIX)
+    execute_process(COMMAND bash -c "git ls-remote https://github.com/KomodoPlatform/atomicdex-desktop-generics refs/heads/main | cut -f 1 | tr -d '\n'"
```

Comment:
This also should be configurable, new company will also fork the desktop-generics repo